javascript/random_alphanumeric.js

```diff
@@ -0,0 +1,4 @@
+var randomAlphaNumeric = (len) => {  for(var rdmString =""; rdmString.length < len; ) rdmString  += Math.random().toString(36).substr(2);  return rdmString.substr(0, len) };
```

Comment:
This solution does not declare a one-line lambda/arrow function and has separate loop/return statements. Maybe you could find a solution with chain-able Array methods?